diff --git a/Formula/php@7.4.rb b/Formula/php@7.4-debug.rb
index c2fffb1..83fb734 100644
--- a/Formula/php@7.4.rb
+++ b/Formula/php@7.4-debug.rb
@@ -1,4 +1,4 @@
class PhpAT74PhpAT74Debug < Formula
  desc "General-purpose scripting language"
  homepage "https://www.php.net/"
  url "https://downloads.php.net/~derick/php-7.4.0RC4.tar.xz"
@@ -100,9 +100,11 @@ class PhpAT74 < Formula
      --with-config-file-scan-dir=#{config_path}/conf.d
      --with-pear=#{pkgshare}/pear
      --with-os-sdkpath=#{MacOS.sdk_path_if_needed}
      --disable-cgi
      --enable-bcmath
      --enable-calendar
      --enable-dba
      --enable-debug
      --enable-dtrace
      --enable-exif
      --enable-ftp
@@ -280,7 +282,7 @@ class PhpAT74 < Formula
  end

  def php_version
    version.to_s.split(".")[0..1].join(".") + "-debug"
  end

  plist_options :manual => "php-fpm"
