diff --git a/Formula/php@7.0.rb b/Formula/php@7.0-debug.rb
index e5169d8..8135feb 100644
--- a/Formula/php@7.0.rb
+++ b/Formula/php@7.0-debug.rb
@@ -1,4 +1,4 @@
class PhpAT70PhpAT70Debug < Formula
  desc "General-purpose scripting language"
  homepage "https://www.php.net/"
  url "https://www.php.net/distributions/php-7.0.33.tar.xz"
@@ -96,6 +96,7 @@ class PhpAT70 < Formula
      --enable-bcmath
      --enable-calendar
      --enable-dba
      --enable-debug
      --enable-dtrace
      --enable-exif
      --enable-ftp
@@ -282,7 +283,7 @@ class PhpAT70 < Formula
  end

  def php_version
    version.to_s.split(".")[0..1].join(".") + "-debug"
  end

  plist_options :manual => "php-fpm"
