diff --git a/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/php.rb b/Formula/php@7.3-debug.rb
index d285abf..d608e3b 100644
--- a/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/php.rb
+++ b/Formula/php@7.3-debug.rb
@@ -1,14 +1,10 @@
class PhpPhpAT73Debug < Formula
  desc "General-purpose scripting language"
  homepage "https://www.php.net/"
  url "https://www.php.net/distributions/php-7.3.11.tar.xz"
  sha256 "657cf6464bac28e9490c59c07a2cf7bb76c200f09cfadf6e44ea64e95fa01021"

  bottle do
    sha256 "c09396d2fc055d74f0f915c475ded38e8f7afac70ca5cc05546629e42fc654f1" => :catalina
    sha256 "1278858c1b7ea78072aa855d2a2de19d6185d000b583787632fb364a4c37157a" => :mojave
    sha256 "98e546809812298562e7059cb2eb6b169672451f809fee53b4cd9cc53b3127ec" => :high_sierra
  endkeg_only :versioned_formula

  depends_on "httpd" => [:build, :test]
  depends_on "pkg-config" => :build
@@ -56,6 +52,10 @@ class Php < Formula
              "APXS_LIBEXECDIR='$(INSTALL_ROOT)#{lib}/httpd/modules'"
      s.gsub! "-z `$APXS -q SYSCONFDIR`",
              "-z ''"

      # apxs will interpolate the @ in the versioned prefix: https://bz.apache.org/bugzilla/show_bug.cgi?id=61944
      s.gsub! "LIBEXECDIR='$APXS_LIBEXECDIR'",
      "LIBEXECDIR='" + "#{lib}/httpd/modules".gsub("@", "\\@") + "'"
    end

    # Update error message in apache sapi to better explain the requirements
@@ -92,9 +92,11 @@ class Php < Formula
      --with-config-file-path=#{config_path}
      --with-config-file-scan-dir=#{config_path}/conf.d
      --with-pear=#{pkgshare}/pear
      --disable-cgi
      --enable-bcmath
      --enable-calendar
      --enable-dba
      --enable-debug
      --enable-dtrace
      --enable-exif
      --enable-ftp
@@ -281,7 +283,7 @@ class Php < Formula
  end

  def php_version
    version.to_s.split(".")[0..1].join(".") + "-debug"
  end

  plist_options :manual => "php-fpm"
