diff --git a/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/php@7.1.rb b/Formula/php@7.1-debug.rb
index 1bee723..c91a731 100644
--- a/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/php@7.1.rb
+++ b/Formula/php@7.1-debug.rb
@@ -1,15 +1,9 @@
class PhpAT71PhpAT71Debug < Formula
  desc "General-purpose scripting language"
  homepage "https://www.php.net/"
  url "https://www.php.net/distributions/php-7.1.33.tar.xz"
  sha256 "bd7c0a9bd5433289ee01fd440af3715309faf583f75832b64fe169c100d52968"

  bottle do
    sha256 "8d5fe6ef069cd38913facb42b72b5597aaeb1d7eb05377ba53259e2d5dd1ba2e" => :catalina
    sha256 "0d349a06ffb681665c2c07efdc33cf9fe5494583024745208cce708ae6b121de" => :mojave
    sha256 "2b3d9af13b1a65c06192ad3a2a8a2ccd06ed4dd2497fe8707c15a6fa812fea5f" => :high_sierra
  end

  keg_only :versioned_formula

  depends_on "httpd" => [:build, :test]
@@ -97,9 +91,11 @@ class PhpAT71 < Formula
      --with-config-file-path=#{config_path}
      --with-config-file-scan-dir=#{config_path}/conf.d
      --with-pear=#{pkgshare}/pear
      --disable-cgi
      --enable-bcmath
      --enable-calendar
      --enable-dba
      --enable-debug
      --enable-dtrace
      --enable-exif
      --enable-ftp
@@ -280,7 +276,7 @@ class PhpAT71 < Formula
  end

  def php_version
    version.to_s.split(".")[0..1].join(".") + "-debug"
  end

  plist_options :manual => "php-fpm"
